### PR TITLE
[Form] fix 'data_class' option in EntityType

### PR DESCRIPTION
**Pull request by HeahDude**
`data_class` option in kind of overridden as it is not used in `EntityType` in favor of required `class` option.

### CODE REVIEW

File: reference/forms/types/entity.rst [hunk 1]
```diff
@@ -21,6 +20,7 @@ objects from the database.
 +-------------+------------------------------------------------------------------+
 | Overridden  | - `choice_list`_                                                 |
 | options     | - `choices`_                                                     |
+|             | - `data_class`_                                                     |
```
Comment (xabbuh) [inline reference/forms/types/entity.rst]: We need to remove two spaces before the last pipe character.

**Comment by xabbuh** [comment]
The first `data_class` description needs to be removed now from the content at the old position (it's now present further down).

**Comment by HeahDude** [comment]
@xabbuh thank you, fixed !

File: reference/forms/types/entity.rst [hunk 2]
```diff
@@ -112,12 +112,10 @@ The class of your entity (e.g. ``AcmeStoreBundle:Category``). This can be
 a fully-qualified class name (e.g. ``Acme\StoreBundle\Entity\Category``)
 or the short alias name (as shown prior).
 
-.. include:: /reference/forms/types/options/data_class.rst.inc
-
 em
 ~~
 
-**type**: ``string`` **default**: the default entity manager
+**type**: ``string`` **default**: the default entity manager class
```
Comment (xabbuh) [inline reference/forms/types/entity.rst]: Well, this option can be a string. But then it is the manager name not the class name. Additionally, you can also pass . `ObjectManager` instances here.

Comment (HeahDude) [inline reference/forms/types/entity.rst]: @xabbuh, thanks, now understood and fixed !

**Comment by xabbuh** [comment]
:+1:

**Comment by WouterJ** [comment]
Thank you Jules.

**Pull request merged**